webapp/src/client/app/shared/interfaces/forms.interfaces.ts

```diff
@@ -40,7 +40,7 @@ export interface VncFormDataInput {
   password: string;
   protocol: number;
   enableCursor: boolean;
-  enableExtendedClipboard: boolean;
+  enableExtendedClipboard?: boolean;
```

Comment:
question: I understand this is unrelated to the main changes? I was left wondering why it was necessary to change this?

Reply:
Done in https://github.com/Devolutions/devolutions-gateway/pull/1473